index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <meta http-equiv="X-UA-Compatible" content="ie=edge">
+      <link rel="stylesheet" href="style.css" type="text/css">
+      <title>Document</title>
+  </head>
+  <body>
+    <div class="wrapper">  
+      <h1>Aleksei Raketskii</h1>
+      <img src="photo.jpg" alt="me" height="400px">
+      <h3>Contacts</h3>
+      <ul>
+        <li><strong>Telephone and messengers(Viber, Telegram, WhatsApp): </strong><a target="_blank"  href="tel:+375295265817">+375295265817</a></li>
+        <li><strong>Email: </strong><a target="_blank"  href="mailto:raketskij@gmail.com">raketskij@gmail.com</a></li>
+        <li><strong>Facebook: </strong><a target="_blank"  href="https://www.facebook.com/profile.php?id=100015919805797">Алексей Ракетский</a></li>
+      </ul>
+      <h3>Summary</h3>
+      <p>
+        From the age of 11 I have been involved in sports programming, so I fancy about programming and everything connected with Information Technologies at all. My positive traits, such as ambitiousness, responsibility and ability to learn quickly allow me to "grasp on the fly".
+      </p>
+      <h3>Skills</h3>
+      <ul>
+        <li> 
+            Programming languages
```

Comment:
As for me, it's better to wrap text into additional tag, e.g. `h4`. Here and below